components/scream/data/screamv1_output.yaml

```diff
@@ -0,0 +1,65 @@
+%YAML 1.1
+---
+Casename: SCREAMv1_output_hydrostatic
+Averaging Type: Instant
+Max Snapshots Per File: 1000
```

Comment:
Maybe we don't care so much about this for this PR, but with 1000 snapshots per file and outputs every 12 steps, won't we get file splits at weird times of day? Would it make more sense to configure to keep 1-file-per-day, or something like that?

Reply:
The 1000 is just the max amount of steps the file can hold before a new one is created.

Reply:
For sure, we could do that. I figured for the analysis we are doing now having one big file is the easiest to visualize